code/modules/client/preference_setup/loadout/loadout_head.dm

```diff
@@ -375,6 +375,21 @@
 	path = /obj/item/clothing/head/hairnet
 	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
 
+/datum/gear/head/deckhelmet
+	display_name = "deck helmet selection"
+	path = /obj/item/clothing/head/deckhelmet
+	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION
```

Comment:
I think you should do away with all the variations and use GEAR_HAS_COLOR_SELECTION instead.

If you are insistent on using the current system, then please use the already estabilished color palette from the existing departmental uniforms.

Reply:
They're meant to be made of brightly colored plastic, but what sort of color hexes should I use for them?